subsys/bluetooth/host/conn.c

```diff
@@ -1895,6 +1922,36 @@ int bt_conn_get_info(const struct bt_conn *conn, struct bt_conn_info *info)
 	return -EINVAL;
 }
 
+#if defined(CONFIG_BT_REMOTE_VERSION)
+int hci_le_read_remote_version(struct bt_conn *conn)
```

Comment:
Move this to `conn.c` as a static function. That way you don't need to touch `hci_core.h`.